packages/core/src/pie-chart.ts

```diff
@@ -232,6 +208,30 @@ export class PieChart extends BaseChart {
 					.style("opacity", 1);
 			});
 
+		// fade out left callout
+		this.innerWrap.select("g.callout-lines-left")
+			.transition()
+			.duration(Configuration.transitions.default.duration / 2)
+			.style("opacity", 0)
+			.on("end", function(d) {
+				select(this)
+					.transition()
+					.duration(Configuration.transitions.default.duration / 2)
+					.style("opacity", 1);
```

Comment:
Same as above

Reply:
Also, I don't think you need `.on("end")` here, you can just chain:
```js
.transition()
	.duration(Configuration.transitions.default.duration / 2)
	.style("opacity", 1);
```

to `.style("opacity", 0)`